doc/classes/RandomNumberGenerator.xml

```diff
@@ -5,6 +5,12 @@
 	</brief_description>
 	<description>
 		RandomNumberGenerator is a class for generating pseudo-random numbers. It currently uses PCG32. The underlying algorithm is an implementation detail. As a result, it should not be depended upon for reproducible random streams across Godot versions.
+		To generate a random float number (within a given range):
+		[codeblock]
+		var rng = RandomNumberGenerator.new()
+		func _process(delta):
+			var my_random_number = rng.randf_range(-10.0, 10.0)
```

Comment:
Indentation should use 4 spaces in code examples (so tabs up to the level of the `[codeblock]`, but further indentation should be with spaces.

Reply:
Ok. So in this case I should remove tabs which push `var my_random_number` and push it by 4 space blocks?